skeleton-common/usr/local/sbin/oc-fetch-ssh-keys

```diff
@@ -39,4 +39,9 @@ EOF
 /usr/local/bin/oc-metadata --cached | grep TAGS_.*=AUTHORIZED_KEY | cut -d'=' -f 3- | sed 's/_/\ /g' >> /root/.ssh/authorized_keys
 
 # Import custom keys
-cat ~/.ssh/instance_keys >> /root/.ssh/authorized_keys
+if [ -f /root/.ssh/instance_keys ]; then
+	cat << EOF >> /root/.ssh/authorized_keys
+# Below your custom ssh keys that you have imported from '/root/.ssh/instance_keys'
```

Comment:
`# Below your custom ssh keys from /root/.ssh/instance_keys` because **we** imported those keys, not **you** :)